bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -308,14 +308,13 @@
 
 	<!-- Litre Channel -->
 	<channel-type id="amrLitre">
-		<item-type>Number:Litre</item-type>
+		<item-type>Number:Volume</item-type>
 		<label>Litre per second</label>
 		<state pattern="%d %unit%" readOnly="true" />
 	</channel-type>
 
-	<!-- Litre Channel -->
 	<channel-type id="amrCubicMetre">
```

Comment:
the id does not fit anymore, but I assume we better keep it for backward compatibility.

Reply:
I would keep the Id for backward compat too.

Reply:
I discussed this with @dominikkv too. We do not believe that anybody uses this thing or these channels currently, so it would be a good chance to change the ids now.

Reply:
ok